src/components/NewCommentForm.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import { Paper } from "@mui/material";
+import { postNewComment } from "../api";
+import { useParams } from "react-router-dom";
+
+const NewCommentForm = ({ comments, setComments }) => {
+  const { article_id } = useParams();
+  const [name, setName] = useState("");
+  const [body, setBody] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setComments(() => {
```

Comment:
Nice execution of a controlled component 👍

Reply:
Thanks Katherine